syntax = "proto3";
package terminus.core;

enum Typology {
  MOTO = 0;
  BIKE = 1;
}


message GeoPoint {
  float lat = 1;
  float lon = 2;
}
